guide/react-native/connecting.mdx

```diff
@@ -28,14 +28,20 @@ const { url, token } = await response.json();
 
 ## Connecting
 
-In order to connect, you just need to call `connect` method with data from previous step:
+In order to connect, you just need to call `joinRoom` method with data from previous step:
 
 ```ts
-import { connect } from "@fishjam-cloud/react-native-client";
+import { joinRoom } from "@fishjam-cloud/react-native-client";
 
-await connect(url, token);
+await joinRoom(url, token);
```

Comment:
**issue (blocking):** joinRoom also need third parameter of ParticipantMetadata.

Reply:
Fixed with this PR: https://github.com/fishjam-cloud/mobile-client-sdk/pull/129